kyo-core/shared/src/test/scala/kyoTest/logsTest.scala

```diff
@@ -9,10 +9,15 @@ class logsTest extends KyoTest:
 
     "log" in IOs.run {
         for
+            _ <- Logs.trace(1 + 1)
             _ <- Logs.trace("trace")
+            _ <- Logs.debug(Seq(1, 2))
             _ <- Logs.debug("debug")
+            _ <- Logs.info(Seq(1, 2), "a")
             _ <- Logs.info("info")
+            _ <- Logs.warn(1, Seq(1, 2), "a")
             _ <- Logs.warn("warn")
+            _ <- Logs.error()
```

Comment:
I thought it'd be good to allow empty params since the log message already contains information of the context. This is handy for debugging.